src/test/java/com/google/api/codegen/testdata/discoveries/mvvm_java/java_adexchangebuyer.v1.4.json.baseline

```diff
@@ -101,12 +91,12 @@ public class AdExchangeBuyerExample {
   }
 
   public AdExchangeBuyer createService() throws IOException, GeneralSecurityException {
-    GoogleCredential credential = GoogleCredential.getApplicationDefault();
-
-    if (credential.createScopedRequired()) {
-      credential =
-          credential.createScoped(Arrays.asList("https://www.googleapis.com/auth/cloud-platform"));
-    }
+    // TODO: Change placeholder below to generate authentication credentials.
+    // See: https://foo.com/bar
+    //
+    // Authorize using one of the following scopes:
```

Comment:
it's possible, i think we'd have to include an `isPlaceholderSingular` function to check and then decide between
`// Authorize using the following scope:`
`// Authorize using one of the following scopes:`

let me know if you want me to do that